docker-compose.override.unit_tests_cicd.yml

```diff
@@ -62,7 +62,28 @@ services:
   redis:
     image: busybox:1.36.1-musl
     entrypoint: ['echo', 'skipping', 'redis']
+
 volumes:
-  defectdojo_data_unit_tests: {}
-  defectdojo_postgres_unit_tests: {}
-  defectdojo_media_unit_tests: {}
+  defectdojo_data_unit_tests:
```

Comment:
Remove these changes. The docker-compose is out of scope.

Reply:
Changes removed